app/src/main/java/org/p2p/wallet/home/ui/main/HomeFragment.kt

```diff
@@ -125,18 +121,34 @@ class HomeFragment :
         imageViewQr.setOnClickListener { replaceFragment(ReceiveSolanaFragment.create(token = null)) }
     }
 
-    private fun ActionButtonsView.setupActionButtons() {
-        onBuyItemClickListener = {
-            presenter.onBuyClicked()
+    private fun LayoutActionButtonsBinding.setupActionButtons() {
+        actionBuyView.apply {
+            textView.setText(R.string.main_buy)
+            imageButton.setImageResource(R.drawable.ic_plus)
+            imageButton.setOnClickListener {
+                presenter.onBuyClicked()
+            }
         }
-        onReceiveItemClickListener = {
-            replaceFragment(ReceiveSolanaFragment.create(token = null))
+        actionReceiveView.apply {
+            textView.setText(R.string.main_receive)
```

Comment:
main -> home

Reply:
no
main buy and receive alreade there
need to rename in future